gitgud/operations.py

```diff
@@ -95,7 +97,9 @@ def create_tree(self, commits, head):
                 for parent in parents[1:]:
                     merge_base = self.repo.merge_base(parents[0], parent)
                     self.repo.index.merge_tree(parent, base=merge_base)
-                self.repo.index.commit(name, author=actor, committer=actor, author_date=committime_rfc, commit_date=committime_rfc, parent_commits=parents)
+                commit_obj = self.repo.index.commit(name, author=actor, committer=actor, author_date=committime_rfc, commit_date=committime_rfc, parent_commits=parents)
+                track_commits(name, commit_obj.hexsha[:7])
+                
 
             commit_objects[name] = self.repo.head.commit
```

Comment:
Do you want to change this to use your new `commit_obj`? It refers to the same thing.

Reply:
bingo